script.js

```diff
@@ -1,12 +1,130 @@
-//You can edit ALL of the code here
+// const allEpisodes = getAllEpisodes();
+
+let allEpisodes = getAllEpisodes();
+// let displayNumOfEpisodes = document.getElementById("numOfEpisodeDisplayed");
+let input = document.getElementById("search");
+let episode = document.createElement("div");
+
+
 function setup() {
   const allEpisodes = getAllEpisodes();
   makePageForEpisodes(allEpisodes);
 }
 
+
 function makePageForEpisodes(episodeList) {
-  const rootElem = document.getElementById("root");
-  rootElem.textContent = `Got ${episodeList.length} episode(s)`;
+  let parent = document.querySelector("#episodeContainer");
+  for (let i = 0; i < episodeList.length; i++) {
+    //for each episode
+    episode = document.createElement("div");
+    let title = document.createElement("h1");
+    let image = document.createElement("img");
+    let summary = document.createElement("p");
+
+    image.setAttribute("src", episodeList[i].image.medium);
+
+    parent.appendChild(episode);
+    episode.appendChild(title);
+    episode.appendChild(image);
+    episode.appendChild(summary);
+
+    episode.className = "card";
+    episode.style.textAlign = "center";
+    title.style.margin = "0 30px 40px 30px";
+    title.style.fontSize = "x-large";
+
+    summary.style.margin = "30px 30px 20px 30px";
+    summary.style.textAlign = "justify";
+
+    // displayNumOfEpisodes.innerText = `Displaying ${episodeList.length}/${allEpisodes.length} episodes`;
+
+    //display title
+    // if (episodeList[i].number < 10) {
+    //   title.innerText = `${episodeList[i].name} - S0${episodeList[i].season}E0${episodeList[i].number}`;
+    // } else {
+    //   title.innerText = `${episodeList[i].name} - S0${episodeList[i].season}E${episodeList[i].number}`;
+    // }
+
+    if (episodeList[i].number < 10) {
+      title.innerText = `${episodeList[i].name} - S0${episodeList[i].season}E0${episodeList[i].number}`;
+    } else {
+      title.innerText = `${episodeList[i].name} - S0${episodeList[i].season}E${episodeList[i].number}`;
+    }
+
+
+    //remove p and br tags
+    let string = episodeList[i].summary;
+    let replaced = string.replace(/(<p>)|(<br>)|(<\/p>)/gm, "");
+    summary.innerText = replaced;
+ }
```

Comment:
Great use of the string method, I did this at the beginning as well, but later discovered that .HTML instead of innerText will sort this without need for replace method, try it.